delta/app/src/test/resources/schemas/errors/invalid-schema.json

```diff
@@ -21,6 +21,6 @@
       },
       "value": "nxv:42"
     },
-    "targetedNodes": 417
+    "targetedNodes": 412
```

Comment:
why is this changing?

Reply:
Not totally sure but we don't load the dash/tosh extensions provided by Topbraid anymore so my guess there are fewer constraints and fewer nodes can get validated in the end.
`targetedNodes` is also a Nexus thing and not sure and a node validated by several constraints will be accounted several times.